src/cms/views/feedback/feedbackList_view.py

```diff
@@ -0,0 +1,129 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+from cms.forms.feedback.feedback import FeedbackForm
+from cms.models.feedback.event_feedback import EventFeedback
+from cms.models.feedback.offer_feedback import OfferFeedback
+from cms.models.feedback.region_feedback import RegionFeedback
+from cms.models.feedback.page_feedback import PageFeedback
+
+
+class FeedbackView(TemplateView):
+    # template_name = "feedback/feedbackList_form.html"
+    template_name = "feedback/feedback_form.html"
+
+    def get(self, request):
+        countUp_event = 0
+        countUp_offer = 0
+        countDown_offer = 0
+        countDown_event = 0
+        countUp_page = 0
+        countDown_page = 0
+        countUp_region = 0
+        countDown_region = 0
+        feedback_events = EventFeedback.objects.filter()
+        feedback_offers = OfferFeedback.objects.filter()
+        feedback_pages = PageFeedback.objects.filter()
+        feedback_regions = RegionFeedback.objects.filter()
+        # event__isnull=False).distinct
+        for event in feedback_events:
+            if event.emotion == "POS":
+                countUp_event = countUp_event + 1
+            else:
+                countDown_event = countDown_event + 1
+
+        for offer in feedback_offers:
+            if offer.emotion == "POS":
+                countUp_offer = countUp_offer + 1
+            else:
+                countDown_offer = countDown_offer + 1
+
+        for page in feedback_pages:
+            if page.emotion == "POS":
+                countUp_page = countUp_page + 1
+            else:
+                countDown_page = countDown_page + 1
+
+        for region in feedback_regions:
+            if region.emotion == "POS":
+                countUp_region = countUp_region + 1
+            else:
+                countDown_region = countDown_region + 1
```

Comment:
```suggestion
        feedback_events = EventFeedback.objects.all()
        feedback_offers = OfferFeedback.objects.all()
        feedback_pages = PageFeedback.objects.all()
        feedback_regions = RegionFeedback.objects.all()

        countUp_event = feedback_events.objects.filter(
            emotion=feedback_emotions.POS
        ).count()
        countDown_event = feedback_events.filter(emotion=feedback_emotions.NEG).count()
        countUp_offer = feedback_offers.filter(emotion=feedback_emotions.POS).count()
        countDown_offer = feedback_offers.filter(emotion=feedback_emotions.NEG).count()
        countUp_page = feedback_pages.filter(emotion=feedback_emotions.POS).count()
        countDown_page = feedback_pages.filter(emotion=feedback_emotions.NEG).count()
        countUp_region = feedback_regions.filter(emotion=feedback_emotions.POS).count()
        countDown_region = feedback_regions.filter(
            emotion=feedback_emotions.NEG
        ).count()
```
This is very inefficient code. Please have a look at the [official Django documentation](https://docs.djangoproject.com/en/2.2/ref/models/querysets/) when you want to extract information from the database. In nearly all cases, there is a cool convenient function from the Django framework to achieve the values you want without any loops etc.